core/src/main/scala/org/http4s/HeaderV2.scala

```diff
@@ -25,8 +25,8 @@ import scala.collection.mutable.{ListBuffer, Set => MutSet}
 
 /** Typeclass representing an HTTP header, which all the http4s
   * default headers satisfy.
-  * You can add custom headers by providing an implicit instance of
-  * `Header[YourCustomHeader]`
+  * You can add modelled headers by providing an implicit instance of
```

Comment:
It is Presidents' Day, yet we still have to endure the British spelling words with two L's! :disappointed:

(No, don't worry about it; let's make it compile.)